test/index.test.ts

```diff
@@ -105,15 +95,12 @@ describe('Configure AWS Credentials', () => {
     jest.spyOn(core, 'info').mockImplementation((string) => {
       return string;
     });
-    (fromEnv as jest.Mock)
-      .mockImplementationOnce(() => () => ({
+    fromEnvMock.mockImplementation(() => () => {
+      return {
         accessKeyId: FAKE_ACCESS_KEY_ID,
         secretAccessKey: FAKE_SECRET_ACCESS_KEY,
-      }))
-      .mockImplementationOnce(() => () => ({
-        accessKeyId: FAKE_STS_ACCESS_KEY_ID,
-        secretAccessKey: FAKE_STS_SECRET_ACCESS_KEY,
-      }));
```

Comment:
there doesn't appear to be any reason to return STS credentials from ENV when called a second time.